docs/framework-introscope_agent.md

```diff
@@ -1,5 +1,5 @@
-# Introscope Agent Framework
-The Introscope Agent Framework causes an application to be automatically configured to work with a bound [Introscope service][].  **Note:** This framework is disabled by default.
+# CA APM Framework
```

Comment:
Since all of the artifacts are still listed as `Introscope` I'd like to have that somewhere in the name of the framework.  I get the need for the new CA branding, but "CA Introscope APM Framework" or something like that is more accurate.  Otherwise we'll need to rebrand the entire component (new config, new Ruby filename, etc.)